src/mod/externals/XMenuTopItem.h

```diff
@@ -0,0 +1,47 @@
+#pragma once
+#include "externals/il2cpp-api.h"
+#include "externals/UnityEngine/MonoBehaviour.h"
+#include "externals/UnityEngine/RectTransform.h"
+#include "externals/UnityEngine/UI/Image.h"
+#include "externals/UnityEngine/Events/UnityAction.h"
+#include "externals/Dpr/UI/UIText.h"
+#include "externals/System/Collections/Generic/List.h"
+#include "externals/System/MulticastDelegate.h"
+namespace UnityEngine {
+    struct XMenuTopItem : ILClass<XMenuTopItem> {
+        struct Fields : UnityEngine::MonoBehaviour::Fields {
+            UnityEngine::RectTransform::Object* _root;
+            UnityEngine::UI::Image::Object* _icon;
+            UnityEngine::UI::Image::Object* _newIcon;
+            UnityEngine::UI::Image::Object* _imageShadow;
+            void* _name;
+            void* _spriteIcons;
+            void* _animator;
+            int32_t _itemType;
+            void* _duplicate;
+        };
+
+    };
+
+    namespace System::Collections::Generic {
+        struct List$$XMenuTopItem : ILClass<List$$XMenuTopItem> {
```

Comment:
You can use the generic List ILClass here, and also same as the previous two lists, no need to redefine the Fields

Reply:
For some reason I had wrapped XMenuTopItem in a UnityEngine namespace and it was stopping List<> from being recognised. Resolved.